tools/SetupFlow/DevHome.SetupFlow/ViewModels/SummaryViewModel.cs

```diff
@@ -205,6 +217,31 @@ protected async override Task OnFirstNavigateToAsync()
             }
         }
 
+        // Collect all next steps.
+        var taskGroups = Orchestrator.TaskGroups;
+        foreach (var taskGroup in taskGroups)
+        {
+            var setupTasks = taskGroup.SetupTasks;
+            foreach (var setupTask in setupTasks)
+            {
+                if (setupTask.SummaryScreenInformation is not null &&
+                    setupTask.SummaryScreenInformation.HasContent)
+                {
+                    switch (setupTask)
+                    {
+                        case CloneRepoTask:
+                            var configResult = new CloneRepoSummaryInformationView();
+                            configResult.DataContext = setupTask.SummaryScreenInformation;
+                            _cloneRepoNextSteps.Add(configResult);
+                            break;

```

Comment:
>break; [](http://example.com/codeflow?start=28&length=6)

how come you break after the first one?  you can only have one next step per taskgroup?

Reply:
`break` here breaks out of the switch statement.  Not the loops. 1:1 between a `setupTask` and `SummaryScreenInformation`